case-app/case
# Add automated tests (server only)

![image](https://user-images.githubusercontent.com/6626184/188428557-a15f4a2c-e837-4a17-b371-984a217309d3.png)


## What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

New feature: It adds automated tests to the CASE application.

## Description

The middle-term goal is to have more tests in the server and to start adding tests in the client. This PR adds 2 kind of automated tests:

- Fixed tests for the CASE app
- An upgrade of the CASE schematics CLI that creates tests for your new resources

Other PRs should follow with more dynamic tests

## Related Issues

None

## How can it be tested?

- *Make sure that you install the 0.4.0 version of the schematics* doing:
```sudo npm i -g @angular-devkit/schematics-cli```
- Install a new CASE instance as you always do
- Once installed, run `npm run test` it should pass all tests on the 2 test files
- Create a resource with `npm run case:resource` and call it as you want. Then run again the `npm run test` command. You should see that there is 2 new testing files for your resources. All tests should pass.


## Check list before submitting
